TypeCobol.Test/Compiler/Parser/ParserUtils.cs

```diff
@@ -134,19 +134,25 @@ public static string DumpCodeElements(CodeElementsDocument result)
 			}
 		}
 
-        internal static string DumpResult(Program program, Class cls, IList<Diagnostic> diagnostics)
+        internal static string DumpResult(IEnumerable<Program> programs, IEnumerable<TypeCobol.Compiler.Nodes.Class> classes, IList<Diagnostic> diagnostics)
         {
             StringBuilder builder = new StringBuilder();
             if (diagnostics != null && diagnostics.Count > 0) {
                 builder.AppendLine(DiagnosticsToString(diagnostics));
             }
-            if (program != null) {
-                builder.AppendLine("--- Program ---");
-                Dump(builder, program);
+            if (programs != null && programs.Count() > 0) {
```

Comment:
Better use .Any().
See https://www.jetbrains.com/help/resharper/2017.1/UseMethodAny.0.html
or you can just check for null